src/main/java/org/launchcode/techjobs/oo/Main.java

```diff
@@ -2,21 +2,46 @@
 
 import java.util.ArrayList;
 
+//public class Main {
+//
+//    public static void main(String[] args) {
+////        Job job1 = new Job("Product tester", new Employer("ACME"), new Location("Desert"), new PositionType("Quality control"), new CoreCompetency("Persistence"));
+////        Job job2 = new Job("Web Developer", new Employer("LaunchCode"), new Location("St. Louis"), new PositionType("Front-end developer"), new CoreCompetency("JavaScript"));
+////        Job job3 = new Job("Ice cream tester", new Employer(""), new Location("Home"), new PositionType("UX"), new CoreCompetency("Tasting ability"));
+////
+////        ArrayList<Job> jobs = new ArrayList<>();
+////        jobs.add(job1);
+////        jobs.add(job2);
+////        jobs.add(job3);
+////
+////        for (Job job : jobs){
+////            System.out.println(job);
+////        }
+//    }
+//
+//}
+import java.util.ArrayList;
+
 public class Main {
 
     public static void main(String[] args) {
-//        Job job1 = new Job("Product tester", new Employer("ACME"), new Location("Desert"), new PositionType("Quality control"), new CoreCompetency("Persistence"));
-//        Job job2 = new Job("Web Developer", new Employer("LaunchCode"), new Location("St. Louis"), new PositionType("Front-end developer"), new CoreCompetency("JavaScript"));
-//        Job job3 = new Job("Ice cream tester", new Employer(""), new Location("Home"), new PositionType("UX"), new CoreCompetency("Tasting ability"));
-//
-//        ArrayList<Job> jobs = new ArrayList<>();
-//        jobs.add(job1);
-//        jobs.add(job2);
-//        jobs.add(job3);
-//
-//        for (Job job : jobs){
-//            System.out.println(job);
-//        }
+        Employer acme = new Employer("ADME");
```

Comment:
Same as the comment regarding moving test code to `JobTest`: https://github.com/KC-CG-Oct-2021/java-assignment-2-techjobs-object-oriented-edition-SavvyRocker666/pull/2/files#r889601862